### DISCUSSION TITLE
Is it possible to give TileLayer or BitmapLayer a lnglat offset to adjust the layer position?

### DISCUSSION THREAD

**Discussion by linze99** [visgl/deck.gl#7415] [Q&A] [answered]
Is it possible to give TileLayer or BitmapLayer a lnglat offset to adjust the tile?
Just like this:
![pic](https://user-images.githubusercontent.com/30024479/201875227-e1dca794-607e-43a1-a0e1-b9b1f6e7782d.PNG)
the yellow road is from mapbox style.
the blue one is from a WMTS layer,
Can i give the layer a [lng, lat] offset  to adjust the layer position?

**Comment by Pessimistress** [comment, marked as answer]
The `BitmapLayer`'s position is controlled by its `bounds` prop.

**Reply by linze99** [reply]
Thanks a lot!
this served my problem.
but after offset the BitmapLayer, the  map  misses some tiles at the edge. 
How can i make the TileLayer to generate more BitmapLayers at the edge?
![1](https://user-images.githubusercontent.com/30024479/202082521-37bd18e7-c245-400a-a3fb-5da56610429c.PNG)

**Reply by MariDani** [reply]
@linze99 Have you solved your issue? I am having similar one..

**Reply by linze99** [reply]
I don't have a perfect solution.
I modify this method: AxisAlignedBoundingBox.intersectPlane; 
then let some edge tiles is INTERSECTION.INSIDE
```js
intersectPlane(plane) {
    const { halfDiagonal } = this;
    const normal = scratchNormal.from(plane.normal);
    const e =
      halfDiagonal.x * Math.abs(normal.x) +
      halfDiagonal.y * Math.abs(normal.y) +
      halfDiagonal.z * Math.abs(normal.z);
    const s = this.center.dot(normal) + plane.distance;

    const differ = s - e;
    const adder = s + e;

    if (adder < -0.1) {
      return INTERSECTION.OUTSIDE;
    }
    if (differ > -0.15) {
      return INTERSECTION.INSIDE;
    }
    if (adder < 0) {
      return INTERSECTION.OUTSIDE;
    }

    return INTERSECTION.INTERSECTING;
    // if (s - e > 0) {
    //   return INTERSECTION.INSIDE;
    // }

    // if (s + e < 0) {
    //   return INTERSECTION.OUTSIDE;
    // }

    // return INTERSECTION.INTERSECTING;
  }
```
